recipe/build.sh

```diff
@@ -1,8 +1,20 @@
 #!/bin/bash
-cp /home/conda/recipe_root/Linux-x86-64-conda.sopt arch/Linux-x86-64-conda.sopt
-make -j${CPU_COUNT} ARCH=Linux-x86-64-conda VERSION=sopt
-make -j${CPU_COUNT} ARCH=Linux-x86-64-conda VERSION=sopt test
+
+# select ARCH file and version
+if [[ "$MACOSX_DEPLOYMENT_TARGET" == "10.9" ]]; then
```

Comment:
wasn't able to find a generic conda variable that just tells the OS platform - if you happen to know one, let me know.
Otherwise one could also check whether this particular variable is non-empty